src/datasets/builder.py

```diff
@@ -724,7 +740,7 @@ def download_and_prepare(
         self._output_dir = fs_token_paths[2][0] if is_local else self._fs.unstrip_protocol(fs_token_paths[2][0])
 
         download_mode = DownloadMode(download_mode or DownloadMode.REUSE_DATASET_IF_EXISTS)
-        verify_infos = not ignore_verifications
+        verification_mode = VerificationMode(verification_mode or VerificationMode.BASIC)
```

Comment:
maybe a stupid question idk but why not `VerificationMode(verification_mode) or VerificationMode.BASIC` (here and everywhere below)?

Reply:
Both approaches work, so I guess this is a matter of style